### ISSUE TITLE
download throttling

### ISSUE THREAD

**Issue by misterhat** [misterhat/livelook#1]
not sure how this could work. the stream throttling module we're using would just store the download data in RAM, so we're really just slowing down how fast it's written to disk. is this all people want with download speed throttling?

**Comment by rudluff** [comment]
my home internet is really bad (about 300kbps up/down most days, sometimes spikes to 1megabye) and if someone on slsk has fast upload it often saturates my connection and results in "frozen" internet for the rest of the family. not a entirely disk write issue!

**Comment by misterhat** [comment]
good point, thanks for the feedback. I get the same issue with bittorrent uploading.
